cmd/internal/flowmode/cmd_run.go

```diff
@@ -157,6 +158,17 @@ func (fr *flowRun) Run(configFile string) error {
 		return fmt.Errorf("building tracer: %w", err)
 	}
 
+	// Enable the "telemetry.useOtelForInternalMetrics" Collector feature gate.
+	// Currently, Collector components uses OpenCensus metrics by default.
+	// Those metrics cannot be integrated with Agent Flow,
+	// so we need to always use OpenTelemetry metrics.
```

Comment:
nit: this removes the TODO comment which mentions that the feature gate will eventually need to be removed once it's removed from OpenTelemetry.